.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+uploads
```

Comment:
There is an extra space before the `uploads` directory. Remove the space to ensure that the `uploads` directory is properly ignored by Git.